sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/OperationScreenBuilder.kt

```diff
@@ -0,0 +1,210 @@
+import br.com.zup.beagle.widget.core.EdgeValue
+import br.com.zup.beagle.widget.core.UnitValue
+import br.com.zup.beagle.widget.layout.Container
+import br.com.zup.beagle.widget.layout.NavigationBar
+import br.com.zup.beagle.widget.layout.Screen
+import br.com.zup.beagle.widget.layout.ScreenBuilder
+import br.com.zup.beagle.widget.layout.ScrollView
+import br.com.zup.beagle.widget.ui.Text
+
+data class Array(val array1: Any, val array2: Any)
+
+object OperationScreenBuilder : ScreenBuilder {
+    override fun build() = Screen(
+        navigationBar = NavigationBar(
+            title = "Operations",
+            showBackButton = true
+        ),
+        child = ScrollView(
+            children = listOf(
+                Container(
+                    context = ContextData("text", "tEsT"),
+                    children = listOf(
+                        stringOperations(),
+                        numberOperations(),
+                        comparisonOperations(),
+                        logicOperations(),
+                        arrayOperations(),
+                        otherOperations()
+                    )
+                ).setStyle {
+                    margin =
+                        EdgeValue(all = UnitValue.real(10))
+                }
+            )
+        )
+    )
+
+    private fun stringOperations() = Container(
+        children = listOf(
+            Text("String", textColor = "#00c91b"),
+            Text(uppercase(expressionOf("@{text}")).capitalize().toBindString()),
+            Text(concat(constant("aaa"), constant("bbb"), expressionOf("@{text}")).toBindString()),
+
+            Text(lowercase(expressionOf("@{text}")).toBindString()),
+            Text((constant("TeStINg".substring(3)).toLowerCase()).toBindString()),
+
+            Text(uppercase(expressionOf("@{text}")).toBindString()),
+            Text(uppercase(expressionOf("@{text}")).toUpperCase().toBindString()),
+
+            Text(substring(constant("testing"), constant(3)).toBindString())
+        )
+    )
+
+    private fun numberOperations() = Container(
+        children = listOf(
+            Container(
+                context = ContextData("number", 4),
+                children = listOf(
+                    Text("Number", textColor = "#00c91b"),
+                    Text(constant("Soma 1 + 2 = ").plus(sum(constant(1), constant(2)).toBindString())),
+                    Text(expressionOf<String>("Soma 1 + 4 = ").toBindString() +
+                            sum(constant(1), expressionOf("@{number}")).toBindString()),
+
+                    Text(sum(constant(1), constant(2), expressionOf("@{number}")).toBindString()),
+                    Text(sum(expressionOf("@{number}"), expressionOf("@{number}")).toBindString()),
+
+                    Text(sum(constant(1), constant(2)).toBindString()),
+                    Text(sum(constant(1),constant(2),sum(constant(2),constant(2),expressionOf("@{number}"))).toBindString()),
```

Comment:
<table data-meta="generated_by_backend">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ZupIT/beagle-backend-kotlin/blob/af6b9a2726483227a5cb2b419f59fc67de6f94de/sample/core/src/main/kotlin/br/com/zup/beagle/sample/builder/OperationScreenBuilder.kt#L121"></span>Line detected that is longer than the defined maximum line length in the code style.</td>
    </tr>
  </tbody>
</table>